core/chains/evm/gas/arbitrum_estimator.go

```diff
@@ -133,6 +134,21 @@ func (a *arbitrumEstimator) GetLegacyGas(ctx context.Context, calldata []byte, l
 	return
 }
 
+// During network congestion Arbitrum's suggested gas price can be extremely volatile, making gas estimations less accurate. For any transaction, Arbitrum will only charge
+// the block's base fee. If the base fee increases rapidly there is a chance the suggested gas price will fall under that value, resulting in a fee too low error.
+// We use gasPriceWithBuffer to increase the estimated gas price by some percentage to avoid fee too low errors. Eventually, only the base fee will be paid, regardless of the price.
+func (a *arbitrumEstimator) gasPriceWithBuffer(gasPrice *assets.Wei, maxGasPriceWei *assets.Wei) *assets.Wei {
+	const gasPriceBufferPercentage = 25
```

Comment:
Thinking more about it.
Could we bump up the buffer to 100%?
It still only requires wallet to hold funds good enough for 2 txs, doesn't sound too much.
we anyways will not pay that, so no harm being generous here.
It will also help avoid Low Fee error more easily.

Reply:
I don't like bumping to 100% because that would mean we completely mask the problem. Since this is an Arbitrum issue, we need to draw the line somewhere and say "Ok, a certain percentage is somewhat understandable, but if the RPC suggests estimations that are more than 25% off, then this should be fixed on their end". Maybe we could reach up to 50% if we feel it's acceptable.